public/lib/helpers/__tests__/timeline_helper.js

```diff
@@ -45,4 +45,26 @@ describe('TimelineHelper', function () {
       expect(timelineHelper.getDescendantPropValue(o, 'p1.p2.p3')).to.eql([7]);
     });
   });
+
+  describe('changeTimezone', function () {
+    it('should return Browser for default Kibana timezone ', function () {
+      expect(timelineHelper.changeTimezone('Browser')).to.be('Browser');
+    });
+
+    it('should return -04:00 for America/Nassau timezone ', function () {
+      expect(timelineHelper.changeTimezone('America/Nassau')).to.be('-04:00');
+    });
+
+    it('should return -13:00 for Etc/GMT-13 timezone ', function () {
+      expect(timelineHelper.changeTimezone('Etc/GMT-13')).to.be('-13:00');
```

Comment:
Ok, so I have to change my code, just a little thing... But this:

> and Etc/GMT+4 should translate to +04:00

should be -04:00, right? :/ sorry but I didn't heard nothing before about _Etc_...